Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -518,13 +518,14 @@
       - ForensicPad
       - ClothingEyesGlassesSecurity
       - RiotShield
-      - CartridgePistol
-      - CartridgeMagnum
       - ShellShotgun
       - ShellShotgunFlare
       - ShellTranquilizer
-      - CartridgeLightRifle
-      - CartridgeRifle
```

Comment:
i think some of these LatheRecipePrototype are never used again, remove em as well if so, also figure out what to do with magnum as its a pretty common weapon

Reply:
No, they're used in the emagged autolathe, the ammo printer (that is not mapped), and presumably some forks and so deserve to stick around.

I readded the magnum ammo box.